proxy/nginx.conf

```diff
@@ -24,6 +24,10 @@ http {
 		location /api/robokache/ {
 			proxy_pass http://robokache/api/;
 		}
+
+		location /api/external/nodeNormalization/ {
+			proxy_pass https://nodenormalization-sri.renci.org/;
```

Comment:
This should probably be a variable, like we did with the frontend url, so it doesn't crash if this host isn't up yet.